### ISSUE TITLE
Questions about the latest SRS workflow

### ISSUE THREAD

**Issue by heather999** [LSSTDESC/ImageProcessingPipelines#44]
Creating a list of issues to ask Nicolas about next week
- [x] is the [srs/scripts](https://github.com/LSSTDESC/ImageProcessingPipelines/tree/master/workflows/srs/scripts) directory obsolete?
- [x] [srs-sub-dev](https://github.com/LSSTDESC/ImageProcessingPipelines/blob/master/workflows/srs/in2p3_vm/srs-sub-dev)
- [x] set header_lines=10 for reference catalog ingest

**Comment by johannct** [comment]
I did not follow closely but the file configs/DC1_ingestRefConfig.py is not the one that I am supposed to use.  @heather can you see what needs to be done here? The 3rd item above can be resolved at the same time. Second point is not to be removed. I am about to remove srs/scripts.

**Comment by heather999** [comment]
Yes, exactly, @johannct, DC1_ingestRefConfig.py is not the correct config anymore, rather we started to use `workflows/srs/dm_configs` to store some configs for the SRS workflow.  I've updated https://github.com/LSSTDESC/ImageProcessingPipelines/blob/u/johannct/%2346/workflows/srs/dm_configs/IngestIndexedReferenceTask_DC2-proto.py to just skip the first 10 lines in the reference catalogs.  In the last couple of months they added 10 comment lines at the top that just need to be ignored. Previously Nicolas and Dominique removed those comments from the reference catalogs to avoid errors when ingesting the reference catalog - which is currently dc2_reference_catalog_dc2v3_fov4_thinned.txt. You might want to check your local copy of that reference catalog to see if it matches the original we have at NERSC in: `/global/projecta/projectdirs/lsst/groups/SSim/DC2/reference_catalogs`
